### PR TITLE
fix: Fix some typos

### PR DESCRIPTION
**Pull request by heitorado**
## What's the purpose of this pull request?
This PR intends to fix typos I've found scattered across the codebase.

## How it works? 
I've run `cspell "**" -e "node_modules" > "spell.txt"` across the codebase. 
- `cspell` is an `npm` package I've installed with `yarn`. 
- The `"**"` arg tells it to search across all codebase.
- The `-e "node_modules"` excludes the `node_modules` folder from the analysis.
- the `> "spell.txt"` saves the output in a file called `spell.txt` which I manually parse and analyze afterwards to search and destroy typos.

## How to test it?
Just renaming variables or editing comments. Nothing should have changed at all.

## References
[cspell](https://www.npmjs.com/package/cspell)
[Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

### CODE REVIEW

File: packages/gatsby-plugin-nginx/src/nginx-generator.ts [hunk 1]
```diff
@@ -217,7 +217,7 @@ export function convertToRegExp(path: string) {
     // allows '<path>/*' to serve '<path>' (without trailing slash)
     // this is necessary because we are now removing trailing slashes from incoming requests
     // so exact matches can work when there is a trailing slash
-    converted = converted.slice(0, -catchAll.length) + '(?:/(.*))?'
+    converted = `${converted.slice(0, -catchAll.length)}(?:/(.*))?`
```
Comment (filipewl) [inline packages/gatsby-plugin-nginx/src/nginx-generator.ts:220]: This is not due to this PR, but what's up with [this ESLint comment](https://github.com/vtex/faststore/runs/5645500405?check_suite_focus=true#step:5:10)? I thought this change would have fixed it.
😕

**Review by filipewl — APPROVED** [review]
Awesome typo fixes!

File: packages/gatsby-plugin-nginx/src/nginx-generator.ts (hunk 1 above)
Comment (eduardoformiga) [inline packages/gatsby-plugin-nginx/src/nginx-generator.ts:220]: this change introduces a Github Actions / Lint warning.
Screen Shot 2022-03-22 at 11 46 01

**Review by eduardoformiga — APPROVED** [review]
~~Sattisfection!~~ I mean, Satisfaction!
Screen Shot 2022-03-22 at 11 50 56

**Review by emersonlaurentino — APPROVED** [review]
Good Job!

File: packages/gatsby-plugin-nginx/src/nginx-generator.ts (hunk 1 above)
Comment (filipewl) [reply packages/gatsby-plugin-nginx/src/nginx-generator.ts:220]: I think he was trying to fix [the previous warning](https://github.com/vtex/faststore/commit/db6f1e173ec43747bf9cfa244b81d86db8a25ce3#diff-ce3b1e791d013aac1413693a4a6e5e04d40451196307f313d8b89a3740cbcf03R220) on this same line but got a different warning this time.

The rule is [prefer-template](https://eslint.org/docs/rules/prefer-template) and the changes make sense on my mind, so I'm not sure why it's still warning.

**Review by saranicoly — APPROVED** [review]
Great job!

**Comment by hellofanny** [comment]
What a great tool! 🚀🚀🚀

**Review by israelswf — APPROVED** [review]
✨ This is magical!

**Pull request merged**